src/shared/App.js

```diff
@@ -6,7 +6,7 @@ import { connect } from 'react-redux';
 import { withNamespaces } from 'react-i18next';
 import { setLocale } from './store/app/actions';
 import Analytics from './pages/analytics';
-import Checkout from './pages/checkout';
+import Checkout from './pages/Checkout';
```

Comment:
Checkout should be lowercase
`import Checkout from './pages/checkout';`
Module not found: Error: [CaseSensitivePathsPlugin] `/Users/armontoya/Documents/MyPrivateFiles/github/cmpe280Project/src/shared/pages/Checkout/index.js` does not match the corresponding path on disk `checkout`.